.travis/install.sh

```diff
@@ -36,6 +36,9 @@ if [[ "$DARWIN" = true ]]; then
             curl -O https://bootstrap.pypa.io/get-pip.py
             sudo python get-pip.py
             ;;
+        bundle)
```

Comment:
Why are you building the mac app with a brew-installed python?  Doesn't this just make the app bundle bigger?  Is the system python somehow inadequate?